solution/backend/serverless-experimental.yml

```diff
@@ -52,6 +52,7 @@ package:
     - nginx/**
 
 custom:
+  envir: 'experimental'
```

Comment:
can we spell out environment for readability?

Reply:
Changed.  I forgot to clean that up.